pmd-apex/src/main/java/net/sourceforge/pmd/lang/apex/ast/ApexTreeBuilder.kt

```diff
@@ -643,6 +680,45 @@ class ApexTreeBuilder(val sourceCode: String, val parserOptions: ApexParserOptio
     private fun buildModifiers(modifiers: List<Modifier>) =
         ASTModifierNode(modifiers).apply { modifiers.forEach { buildAndSetParent(it, parent = this) } }
 
+    /** Generates additional nodes for the [root] node. */
+    private fun generateAdditional(root: ApexNode<*>) {
```

Comment:
Suggest longer name

Reply:
```kotlin
/** Generates additional nodes under the [root] node in a second pass after the initial [construction][build] of the AST. */
private fun generateAdditionalNodes(root: ApexNode<*>) {
```
Is this more clear? I seem to have trouble documenting these kinds of functions..